GitCommands/GitRevisionInfoProvider.cs

```diff
@@ -14,7 +15,7 @@ public interface IGitRevisionInfoProvider
         /// </summary>
         /// <param name="item"></param>
         /// <returns>The item's children.</returns>
-        IEnumerable<IGitItem> LoadChildren(IGitItem item);
+        Task<IEnumerable<IGitItem>> LoadChildren(IGitItem item);
```

Comment:
❗️ Missing `Async` suffix for the method name

Reply:
Thanks, will fix now.